src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -403,6 +435,17 @@ private static List<List<T>> Transpose<T>(List<List<T>> columns, int columnSize)
                     resultRows.Add(DValue<RecordValue>.Of(record));
                 }
 
+                // Add error nodes for different table length
+                // e.g. Concatenate(["a"],["1","2"] => ["a1", <error>]
+                var namedErrorValue = new NamedValue(columnNameStr, FormulaValue.NewError(new ExpressionError()
+                {
+                    Kind = ErrorKind.NotSupported,
```

Comment:
>NotSupported [](http://example.com/codeflow?start=37&length=12)

We should have a new error kind that maps to Excel's `#N/A` - like "ErrorKind.NotApplicable"